labelbox/client.py

```diff
@@ -52,6 +53,11 @@ def __init__(self,
             api_key = os.environ[_LABELBOX_API_KEY]
         self.api_key = api_key
 
+        if verbose:
```

Comment:
verbose is now an arg that will help specify which logger the logger is. lmk if this is the right way to do this!

Reply:
i'd suggest something like:


```suggestion
        if verbose:
            logger.setLevel(level=logging.INFO)
```

and then where you declare `logger` in `lb_logger` just have it be a `logging.getLogger` call

Reply:
something I noticed is that .setLevel is not sufficient to change the logging output. maybe instead of 
`logger.setLevel(level=logging.INFO`
we can go with
`logging.basicConfig(level=logging.INFO)`

since this originated from `bulk_import_request` not being `level=logging.INFO`, what is the best way to have it so that that file will inherit if verbose == True?

Reply:
hmm setLevel should be sufficient as i understand it

and since we're using the same logger throughout, BIR should end up using the logger after the client has changed the level

Reply:
I see now, thanks for explaining BIR for me

setLevel unfortunately does not looks like it works. can provide an example file which indicates this

Reply:
hmm. will relook at this. test seems to show it works correctly but an external example test shows it does not